types/types.go

```diff
@@ -160,3 +161,47 @@ func (m ComMatrix) Contains(cd ComDetails) bool {
 
 	return false
 }
+
+func ToNFTables(m ComMatrix) ([]byte, error) {
+	var tcpPorts []string
+	var udpPorts []string
+	for _, line := range m.Matrix {
+		if line.Protocol == "TCP" {
+			tcpPorts = append(tcpPorts, fmt.Sprint(line.Port))
+		} else if line.Protocol == "UDP" {
+			udpPorts = append(udpPorts, fmt.Sprint(line.Port))
+		}
+	}
+
+	tcpPortsStr := strings.Join(tcpPorts, ", ")
+	udpPortsStr := strings.Join(udpPorts, ", ")
+
+	result := fmt.Sprintf(`#!/usr/sbin/nft -f
+
+	table ip filter {
```

Comment:
The same table name is created by the cluster already. It is highly advisable to use a different name.